go.mod

```diff
@@ -38,6 +38,7 @@ require (
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/emicklei/go-restful/v3 v3.11.0 // indirect
+	github.com/evanphx/json-patch v5.6.0+incompatible // indirect
```

Comment:
Why do we need this change?

Reply:
No idea. To be able to run the tests locally, I had to run `go mod tidy` and it added that.
Not sure how to avoid it.